java/code/src/com/suse/manager/reactor/utils/test/RhelUtilsTest.java

```diff
@@ -288,6 +303,17 @@ public void testDetectRhelProductAmazon() throws Exception {
                 });
     }
 
+    public void testDetectRhelProductRocky() throws Exception {
+        doTestDetectRhelProduct("dummy_packages_redhatprodinfo_rockylinux.json",
+                null,
+                prod -> {
+                    assertFalse(prod.get().getSuseProduct().isPresent());
```

Comment:
I think this should change to:
```
                    assertTrue("SUSE Product not found", prod.get().getSuseProduct().isPresent());
```
As soon as we have SCC products and the definitions as we did for Alma at https://github.com/uyuni-project/uyuni/commit/915d51c0a5e8ab434ace2310714b71ccb7d256df

Reply:
:+1: